docs/Catalog/PackageDialog.md

```diff
@@ -0,0 +1,58 @@
+# Opening package dialog via a URL
+
+You can use the `createPackage` query parameter to create a URL that opens
+the Quilt Catalog directly to the package dialog with certain information
+already filled in.
+
+Available parameters are
+
+* `createPackage=true` to open a package dialog,
```

Comment:
i'd prefer some shorter lowercase params like `edit=` and `dropzone=`, but i guess it would be more ambiguous, so whatever